code/modules/mob/living/living.dm

```diff
@@ -1386,3 +1388,39 @@
 	if(is_servant_of_ratvar(src) && !iseminence(src))
 		eminence.selected_mob = src
 		to_chat(eminence, "<span class='brass'>You select [src].</span>")
+
+
+/mob/living/ghostize(can_reenter_corpse = TRUE, retain_sentience = FALSE )
+	..()
+	if(retain_sentience)
+		set_playable()
+
+/mob/living/attack_ghost(mob/user)
+	. = ..()
+	if(.)
+		return
+	give_mind(user)
+
+/mob/living/Topic(href, href_list)
+	if(href_list["activate"])
+		var/mob/dead/observer/ghost = usr
+		if(istype(ghost) && playable)
+			give_mind(ghost)
```

Comment:
You need to return if ..() I believe.
In any case ..() will definitely need to be called so the mob topics work assuming they do work. Have a look at other topics and see how its written out but I think it should be if(..()) return or something

Reply:
You were right.

The function now concludes with '..()'